src/stan/model/indexing/rvalue.hpp

```diff
@@ -100,7 +119,7 @@ inline T rvalue(
 /**
  * Return a single element of an Eigen Vector.
  *
- * Types:  vec[uni] : scal
+ * Types:  vector[uni] : scaler
```

Comment:
```suggestion
 * Types:  vector[uni] : scalar
```

Reply:
Also the docs are inconsistent. Somethine `uni` is used and sometimes `single`.

Reply:
Word I'll fix this up!